bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/ButtonImpl.java

```diff
@@ -81,6 +81,7 @@ public String getDefault() {
         return defaultValue;
     }
 
+    @JsonIgnore
```

Comment:
why do we need to make changes in Button's model?

Reply:
In Button Model, because the schema right now, doesn't allow properties buttonType, and type, but the model.json created had these, so I made this change.